src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -332,7 +332,7 @@ int mongoSubCacheItemInsert
 
     if (!entity.hasField(CSUB_ENTITY_ID))
     {
-      LM_W(("Runtime Error (got a subscription without id)"));
+      LM_E(("Runtime Error (got a subscription without id)"));
```

Comment:
Subscription without id shouldn't occur at CB. Worth to be ERROR instead of WARN.

NTC (informative)